### PR TITLE
Social verification

### PR DESCRIPTION
**Pull request by tibetsprague**
Add social verification buttons to account profiles 
Github, Twitter and Facebook for now
Always show Edit Profile link in the header
Linter now requires semicolons everywhere

### CODE REVIEW

**Comment by jellegerbrandy** [comment]
I approved this bug realize there is still a bug. EVen without alchemy-server running, on the profile page, I can submit (and sign) and will get no feedback at all. 
The bug is in this line: https://github.com/daostack/alchemy/blob/social-verification/src/actions/profilesActions.ts#L90

Because in this case, `e.response.data` is undefined, so it throws on this line and that error is not caught...

File: src/actions/arcActions.spec.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { getDAOs, createDAO, CreateDAOAction, createProposal, CreateProposalActi
 import { Dispatch } from "react-redux";
 import { IRootState } from "reducers";
 import * as Arc from "@daostack/arc.js";
-import { ActionTypes } from "reducers/arcReducer"
+import { ActionTypes } from "reducers/arcReducer";
```
[the remark this answers is not in the document]
Comment (tibetsprague) [reply src/actions/arcActions.spec.ts]: Well lets have that conversation. I lean towards semicolons myself...

**Comment by tibetsprague** [comment]
Yes I will definitely push to staging for a full test

File: src/actions/arcActions.spec.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (tibetsprague) [reply src/actions/arcActions.spec.ts]: https://hackernoon.com/an-open-letter-to-javascript-leaders-regarding-no-semicolons-82cec422d67d

**Pull request merged**